src/settings_gui.cpp

```diff
@@ -140,6 +142,46 @@ void ShowBaseSetTextfileWindow(TextfileType file_type, const TBaseSet* baseset,
 	new BaseSetTextfileWindow<TBaseSet>(file_type, baseset, content_type);
 }
 
+std::vector<int> _refresh_rates = { 30, 60, 75, 90, 100, 120, 144, 240 };
+
+/* We want 60Hz as the default for this pointer, though it gets overriden with whatever is in the config. */
+int *_current_refresh_rate = &_refresh_rates[1];
+
+static const std::string HERTZ = "Hz";
+std::string _current_refresh_rate_str = std::to_string(*_current_refresh_rate) + HERTZ;
```

Comment:
I feel like this is better off being an (inline?) function, rather than a global that constantly needs to be (remembered to be) updated